tests/common.go

```diff
@@ -2060,15 +2389,18 @@ func SetSourceKubeConfig() error {
 	if err != nil {
 		return err
 	}
-	SetClusterContext(sourceClusterConfigPath)
-	return nil
+	err = SetClusterContext(sourceClusterConfigPath)
+	return err
 }
 
 // SetDestinationKubeConfig sets current context to the kubeconfig passed as destination to the torpedo test
-func SetDestinationKubeConfig() {
+func SetDestinationKubeConfig() error {
 	destClusterConfigPath, err := GetDestinationClusterConfigPath()
-	expect(err).NotTo(haveOccurred())
-	SetClusterContext(destClusterConfigPath)
+	if err != nil {
```

Comment:
How about copying the `expect(err).NotTo(haveOccurred())` assertion from SetDestinationKubeConfig to SetSourceKubeConfig too? This would ensure that the test case stops execution if the assertion fails, which is expected since neither SetDestinationKubeConfig nor SetSourceKubeConfig did not handle errors before.

Reply:
I made both functions uniform so that they return errors so that we can handle them ourselves (in the testcase) as we don't want to assume that failure to switch context should definitely fail the test.

Reply:
@mkoppal-px / @lsrinivas-pure / @kshithijiyer-px please advice whether to handle the error ourselves or terminate the execution immediately.